ci/workflows/smoke.workflow

```diff
@@ -19,31 +19,17 @@ action "download test data" {
   runs = ["sh", "-c", "ci/scripts/download-test-data.sh"]
 }
 
-# NOTE: done offline to speedup travis build time
-#
-# action "build ceph image" {
-#   needs = "download test data"
-#   uses = "actions/docker/cli@master"
-#   args = "build -t popperized/ceph:luminous ci/docker"
-# }
-#
-# action "registry login" {
-#   needs = "build ceph image"
-#   uses = "actions/docker/login@master"
-#   secrets = ["DOCKER_USERNAME", "DOCKER_PASSWORD"]
-# }
-#
-# action "push image" {
-#   needs = "registry login"
-#   uses = "actions/docker/cli@master"
-#   args = "push popperized/ceph:luminous"
-# }
+action "build ceph image" {
```

Comment:
can you just add a quick comment indicating this is building from main ceph-luminous, not skyhook repo correct?